### DISCUSSION TITLE
Can't use the fork activity in Elsa 2

### DISCUSSION THREAD

**Discussion by riatdyl** [elsa-workflows/elsa-core#934] [Q&A]
Hello,

I'm trying Elsa 2 and I have a problem.

In Elsa 1, the fork activity is working and makes me 2 outcomes.

In the Elsa 2 Workflow, I can place a fork activity and give it two branches. The problem is that this does nothing.

![image](https://user-images.githubusercontent.com/43779018/117139838-ff0db280-adac-11eb-86c0-c0829a6ceaab.png)
![image](https://user-images.githubusercontent.com/43779018/117139882-0634c080-adad-11eb-8d32-242d09f7a969.png)

**Comment by sfmskywalker** [comment]
This was indeed an issue which has been fixed since. If you update to the latest version it will just work.
